site/src/docs/components/file-input/index.mdx

```diff
@@ -96,3 +96,28 @@ export const FileInputDragAndDrop = () => {
 <PlaygroundPreview>
   <FileInputDragAndDrop />
 </PlaygroundPreview>
+
+#### With default value
+
+If needed, the file input can be prefilled with pre-existing file(s). Useful, for example, if adding files in a multi-page form and coming back to the file-adding step again.
+
+export const FileInputDefaultValue = () => {
+  const [files, setFiles] = React.useState([
+    new File(['string content'], 'dummy.txt', { type: 'text/plain' }),
```

Comment:
Causes error on documentation gatsby build `WebpackError: ReferenceError: File is not defined` 🤔

Reply:
Ah yeah the SSR 🤦

Reply:
This appears to be fixed, resolving 👍